feat(examples): add deadline secrets management options to basic example (#562)
aws/aws-rfdk: examples/deadline/All-In-AWS-Infrastructure-Basic/python/README.md

    ```python
    self.alarm_email_address: Optional[str] = 'username@yourdomain.com'
    ```

> 15. By default Secret Management is enabled but it can be disabled by chnging `enable_secret_management` variable in `package/config.py`

Reviewer: Maybe we can explain a little more about the feature and point to the readme which is being updated [here](https://github.com/aws/aws-rfdk/pull/558/files#diff-1a6bfdf8589ae41c676e19bd04925147c6140c074cf28f1cb4c61ec9b7063cc5). This would also apply to the typescript readme.

```suggestion
15. Deadline Secrets Management is a feature used to encrypt certain values in the database that need to be kept secret. Additional documentation about the feature and how it works in the RFDK can be found in the [RFDK README](../../../../packages/aws-rfdk/lib/deadline/README.md). By default, Deadline Secrets Management is enabled, but it can be disabled by changing the `enable_secret_management` variable in `package/config.py`.
```

> 16. When you are using Secret Management you can defined own admin credentials by creating secret in SecretManager in format:

Reviewer: Let's give the password requirements so they don't have to dig into our documentation for them. This also applies to the typescript readme.

```suggestion
16. When you are using Deadline Secrets Management you can define your own admin credentials by creating a Secret in SecretsManager in the following format. The password must be at least 8 characters long and contain at least one lowercase, one uppercase, one digit, and one special character.
```

    ```python
    self.alarm_email_address: Optional[str] = 'username@yourdomain.com'
    ```

> 15. Deadline Secrets Management is a feature used to encrypt certain values in the database that need to be kept secret. Additional documentation about the feature and how it works in the RFDK can be found in the [RFDK README](../../../../packages/aws-rfdk/lib/deadline/README.md). By default, Deadline Secrets Management is enabled, but it can be disabled by changing the `enable_secret_management` variable in `package/config.py`.

Reviewer: Let's add the code example here like we did in the typescript readme.

Reviewer 2: ```suggestion
15. Deadline Secrets Management is a feature used to encrypt certain values in the database that need to be kept secret. Additional documentation about the feature and how it works in the RFDK can be found in the [RFDK README](../../../../packages/aws-rfdk/lib/deadline/README.md). By default, Deadline Secrets Management is enabled, but it can be disabled by changing the `enable_secrets_management` variable in `package/config.py`.
```

Let's rename the variable to be consistent with "Deadline Secrets Management". Same suggestion for other places we use `secret_management` in this PR

16. When you are using Deadline Secrets Management you can define your own admin credentials by creating a Secret in SecretsManager in the following format:

    ```json
        {
            "username": <admin user name>,
            "password": <admin user password>,
        }
    ```
    The password must be at least 8 characters long and contain at least one lowercase, one uppercase, one digit, and one special character.

    Then the value of the `secret_management_secret_arn` variable in `package/config.py` should be changed to this secret's ARN:

    ```python
    self.ubl_certificate_secret_arn: str = '<your secret arn>'
    ```

> 

Reviewer: nit: Extra blank line

    ```python
    self.alarm_email_address: Optional[str] = 'username@yourdomain.com'
    ```
15. Deadline Secrets Management is a feature used to encrypt certain values in the database that need to be kept secret. Additional documentation about the feature and how it works in the RFDK can be found in the [RFDK README](../../../../packages/aws-rfdk/lib/deadline/README.md). By default, Deadline Secrets Management is enabled, but it can be disabled by changing the `enable_secret_management` variable in `package/config.py`.

    ```python

>     self.alarm_email_address: Optional[str] = 'username@yourdomain.com'

Reviewer 2: ```suggestion
    self.enable_secrets_management: bool = False
```

    ```

> 16. When you are using Deadline Secrets Management you can define your own admin credentials by creating a Secret in SecretsManager in the following format:

Reviewer: ```suggestion
16. When you are using Deadline Secrets Management you can define your own admin credentials by creating a Secret in AWS SecretsManager in the following format:
```

    ```json
        {

>             "username": <admin user name>,
>             "password": <admin user password>,

Reviewer: I just realized the reason these values are highlighted red in the review is because they are missing quotes around them, making them invalid JSON. The rich diff also shows they are highlighted in red, so I think maybe we should fix it now to make sure that it won't look like that when it's committed:
```suggestion
            "username": "<admin user name>",
            "password": "<admin user password>",
```

        }
    ```
    The password must be at least 8 characters long and contain at least one lowercase, one uppercase, one digit, and one special character.

>     Then the value of the `secret_management_secret_arn` variable in `package/config.py` should be changed to this secret's ARN:

Reviewer: ```suggestion
    Then the value of the `secrets_management_secret_arn` variable in `package/config.py` should be changed to this Secret's ARN:
```

    ```python

>     self.enable_secret_management: bool = False

Reviewer 2: ```suggestion
    self.secrets_management_secret_arn: Optional[str] = <your secret arn>
```
